### PR TITLE
Reorganize the files for NeuroQuery/NiMARE compatibility

### LINKED ISSUE #5: Reorganize files into npz + txt files to match NeuroQuery
[NeuroQuery](https://github.com/neuroquery/neuroquery_data) organizes its feature data into (1) npz files for the weights, (2) txt files for the columns, and (3) txt files for the study IDs (the index). This should reduce the size of the dataset, and should make it easier to ingest both NeuroQuery and Neurosynth data into NiMARE and NIMADS formats.

### PR DESCRIPTION
**Pull request by tsalo**
Closes #5. This increases the overall size from 246M to 387M.

Changes proposed:

- Rename files according to the naming convention proposed in https://github.com/neuroquery/neuroquery_data/issues/3.
- Split feature files into IDs/vocabulary/weights triplets.

### CODE REVIEW

**Comment by tyarkoni** [comment]
LGTM but can you verify that the description of the data in the README still makes sense, and update it if not?

File: README.md [hunk 1]
```diff
@@ -1,36 +1,78 @@
...
 
-* archive/data_0.5.February_2015.tar.gz: The format and structure of all files in this release is identical to the previous one (0.4); the only difference is the addition of new studies and features. The 0.5 release contains data for > 10,900 studies and > 3,300 features.
+- `vocab-terms`: This vocabulary refers to terms extracted from abstracts using a vectorizer.
+  Prior to version 0.5, these terms were simply individual words.
+  However, from version 0.5 on, these terms have included both individual words (unigrams) and two-word terms (bigrams).
+  Terms are extracted on a largely unsupervised basis; namely, some words are *not* extracted, based on a "stop list",
+  while others may appear too frequently (e.g., common words like "the") or too infrequently to be meaningful.
+  The current version's terms vocabulary is what is accessed on the Neurosynth website.
+  Additionally, the values for the `features.npz` for this vocabulary are [tf-idf](https://en.wikipedia.org/wiki/Tf–idf) values, starting at version 0.4.
+  In versions prior to 0.4, these values were proportions.
```
Comment (tsalo) [inline README.md:52]: I just want to make sure this is accurate. If so, I plan to rename the `type-tfidf` files from version 0.3 to `type-proportion`.

**Comment by tsalo** [comment]
Thanks @tyarkoni! You're right- I forgot to update the README, but now it should be good.

Two pending things though:

1. I just want to make sure it's cool that I dropped 0.2. The files were formatted differently and it seemed like fixing them would be more effort than it was worth.
2. Version 0.4 is where you switched from proportion weights to tf-idf, right?

**Comment by tyarkoni** [comment]
On (1), agreed. On (2), I don't remember, and don't have time to reconstruct. Seems unlikely anyone will ever want to use that data though, so I wouldn't sweat it. Thanks for doing this! Feel free to merge whenever you're done.

**Comment by tsalo** [comment]
Thanks! I'll hold off until neuroquery/neuroquery_data#3 is resolved (I don't want to have to change the entity order or anything in a new PR), and then I'll merge.

**Pull request merged**